### DISCUSSION TITLE
ScriptReference/AI.NavMesh.CalculateTriangulation

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#222] [Notes]
# ScriptReference/AI.NavMesh.CalculateTriangulation

https://docs.unity3d.com/ScriptReference/AI.NavMesh.CalculateTriangulation.html

**Comment by mvi** [comment]
Note that the mesh produced does not appear to be completely deterministic and when testing with 2019.3.0b10 produces different results even with sequential CalculateTriangulation calls. In my tests, the triangulation appears to deterministic but alternates between two different triangulations with each call.
